src/style.css

```diff
@@ -0,0 +1,18 @@
+body {
+  margin: 0;
+  padding: 0;
```

Comment:
они и так 0

Reply:
Your browser will add a narrow margin to the `<body>` element by default. Remove it so that there is no space between the edges of the page and the blocks.

Reply:
Речь о паддингах, они у браузера по-умолчанию 0.